modules/standard/Random.chpl

```diff
@@ -915,6 +920,7 @@ module Random {
         :type n: `integral`
 
         :returns: The `n`-th value in the random stream as type :type:`eltType`.
+        :throws IllegalArgumentError: When called with non-positive 'n' value.
```

Comment:
I'm not going to call out every case but this is to point out that there are more instances than just the one